src/app/components/LiteSiteCta/index.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render } from '../react-testing-library-with-providers';
+import LiteSiteCta from '.';
+
+describe('LiteSiteCTA', () => {
+  it('Should have a hidden strong element with lite site identifier.', () => {
+    const { container } = render(<LiteSiteCta />);
+    const strongText = container.querySelector('strong');
+    expect(strongText?.innerHTML).toBe('Data saving version');
+    expect(strongText).toHaveAttribute('hidden');
+  });
+  it('Should have a CTA link to the main site.', () => {
+    const { container } = render(<LiteSiteCta />);
+    const ctaText = container.querySelectorAll('a span')[0];
+    const ctaLink = container.querySelectorAll(
+      'a[href="https://www.test.bbc.com/news/articles/c0g992jmmkko"]',
+    )[0];
```

Comment:
We can use array destructing here
```suggestion
    const [ctaText] = container.querySelectorAll('a span');
    const [ctaLink] = container.querySelectorAll(
      'a[href="https://www.test.bbc.com/news/articles/c0g992jmmkko"]',
    );
```

Reply:
Nice one, thanks!